io.catenax.single_level_bom_as_built/3.0.0/SingleLevelBomAsBuilt.ttl

```diff
@@ -0,0 +1,105 @@
+   samm:properties (:catenaXId :childItems);
+   samm:operations ();
+   samm:events ().
+
+:catenaXId a samm:Property;
+   samm:preferredName "Catena-X ID"@en;
+   samm:description "The Catena-X ID of the given part (e.g. the assembly), valid for the Catena-X dataspace."@en;
+   samm:characteristic ext-uuid:UuidV4Trait;
+   samm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d".
+
+:childItems a samm:Property;
+   samm:preferredName "Child Items"@en;
+   samm:description "Set of child items, of which the given parent item is assembled by (one structural level down)."@en;
+   samm:characteristic :SetOfChildItemsCharacteristic.
+
+:SetOfChildItemsCharacteristic a samm-c:Set;
+   samm:preferredName "Set of Child Items"@en;
+   samm:description "Set of child items the parent item is assembled by (one structural level down)."@en;
+   samm:dataType :ChildData.
+
+:ChildData a samm:Entity;
+   samm:preferredName "Child Data"@en;
+   samm:description "Catena-X ID and meta data of the assembled child item."@en;
+   samm:properties (:createdOn :quantity [ samm:property :lastModifiedOn; samm:optional true ] :businessPartner :hasAlternatives :childCatenaXId).
+
+:createdOn a samm:Property;
+   samm:preferredName "Created On"@en;
+   samm:description "Timestamp when the relation between the parent item and the child item was created, e.g. when the serialized child part was assembled into the given part."@en;
+   samm:characteristic samm-c:Timestamp;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+
+:quantity a samm:Property;
+   samm:preferredName "Quantity"@en;
+   samm:description "Quantity of which the child item is assembled into the parent item. In general it is '1' for serialized parts."@en;
+   samm:characteristic :QuantityCharacteristic.
+
+:lastModifiedOn a samm:Property;
+   samm:preferredName "Last Modification Date"@en;
+   samm:description "Timestamp when the assembly relationship between parent item and child item was last modified."@en;
+   samm:characteristic samm-c:Timestamp;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+
+:businessPartner a samm:Property;
+   samm:preferredName "Business Partner"@en;
+   samm:description "The supplier of the given child item."@en;
+   samm:characteristic ext-number:BpnlTrait;
+   samm:exampleValue "BPNL50096894aNXY".
+
+:hasAlternatives a samm:Property;
+   samm:preferredName "Has Alternatives"@en;
+   samm:description "Expresses wether the part is built-in or wether it is one of several options. If the value is false, it can be assumend this exact item is built-in. If the value is true, it is unknown wether this or an alternative item is built-in.\nThis is the case when, e.g. the same item is supplied by two suppliers, the item is only tracked by a customer part ID during assembly. Thus, these items can not be differentiated from each other.\n\n"@en;
+   samm:characteristic samm-c:Boolean.
+
+:childCatenaXId a samm:Property;
```

Comment:
want to rename this property or is it correct?
Also, in preferred name it is written 'Catena-X Parent ID'. please correct.